CodeFirstStoreFunctionsTests/StoreFunctionBuilderTests.cs

```diff
@@ -55,13 +55,16 @@ public void Crate_creates_store_function_for_complex_type_function_import()
             var model = new DbModelBuilder()
                     .Build(new DbProviderInfo("System.Data.SqlClient", "2012"));
 
+            var enumType = EnumType.Create("TestEnum", "TestNs",PrimitiveType.GetEdmPrimitiveType(PrimitiveTypeKind.Int32), false, new EnumMember[]{EnumMember.Create("foo", 1, null)}, null);

```

Comment:
nit: formatting (add a whitespace after `"TestNs",` and fix array initialization formatting). Do it as you did in this line: https://github.com/Amberg/CodeFirstFunctions/blob/e2ce15e5c1c3746becaa1eed87f866b7e93402c6/CodeFirstStoreFunctionsTests/StoreFunctionBuilderTests.cs#L99